stream-chat-android/src/main/res/values/attrs.xml

```diff
@@ -178,7 +178,7 @@
         <attr name="streamSendAlsoToChannel" format="color|reference" />
     </declare-styleable>
     <!--MessageListView-->
-    <declare-styleable name="MessageListView">
+    <declare-styleable name="StreamMessageListView">
```

Comment:
Looks like the layout editor relies on the name of the styleable matching the name of the View to show suggestions. I don't get any autocompletion results when I try to add attributes (for example, the newly added ones) to the `MessageListView` in `fragment_channel.xml` after this change.

I think we should remove prefixes from all of our styleables, as our Views don't have them, and then suppress the warning about their names, if possible.

Reply:
It's reverted now. Let's clean it up in another PR.